docs/developer/sql-test-environment/README.md

```diff
@@ -0,0 +1,60 @@
+## Local setup for SQL query testing
+
+### Requirements
+
+- Docker
+- HTTP Client of your choice
+- Access to ghcr.io with permissions for Aam Digital private repositories
+
+### Start containers
+
+Take a look into the `docker-compose.yml` and adapt the file if necessary.
+
+If you're on an ARM based system, you need to enable for the SQS container:
+
+- `platform: linux/amd64`
+
+#### Authorization ghcr.io
+
+You will need to do a `docker login` for ghcr.io, to be able to fetch sqs image.
+
+See [GitHub Documentation](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry) for details.
+
+### Access CouchDb web interface
+
+You can access the CouchDb webinterface here: http://localhost:5984/_utils/
+
+The default credentials are:
+
+username: `admin`  
+password: `docker`
+
+### Using existing Database
+
+If you have a copy from an existing database, stop the couchdb container and just copy all files from the `couchdb/data` directory
```

Comment:
This will only help people who have access to a specific server. Did you try to use the sync UI of CouchDB Fauxton? This sometimes worked for me, sometimes I didn't get it to run :see_no_evil:

Reply:
correct, i never used the sync ui for that... but it could work.